ffs/internal/cmdblob/commands.go

```diff
@@ -290,12 +249,14 @@ func readData(ctx context.Context, cmd string, args []string) (data []byte, err
 	return
 }
 
-func storeFromEnv(env *command.Env) (context.Context, config.CAS, error) {
-	t := env.Config.(*config.Settings)
-	bs, err := t.OpenStore(env.Context())
-
-	// Because the blob commands operate on the raw store, take off the default
-	// data bucket suffix and apply the one from the -bucket flag.
-	bs.CAS = bs.CAS.WithSuffix(blobFlags.Bucket)
-	return env.Context(), bs, err
+func withStoreFromEnv(env *command.Env, f func(blob.CAS) error) error {
+	bs, err := env.Config.(*config.Settings).OpenStore(env.Context())
+	if err != nil {
+		return err
+	}
+	raw, err := bs.Store().Keyspace(env.Context(), blobFlags.Bucket)
```

Comment:
I think this .Keyspace() is supposed to be .Sub() ? At least bs.Store() returns a StoreCloser and I don't see a Keyspace method anywhere. I suspect it might be fixed in a later commit in the series?

Reply:
I think this must be from a previous commit, probably I did not squash my crimes fully. It is calling `KV` at the tip of the branch.